cmd/root.go

```diff
@@ -125,68 +125,31 @@ func fileRelative(p string) (string, error) {
 }
 
 func createFn(cmd *cobra.Command, args []string) error {
-	topopb, err := topo.Load(args[0])
-	if err != nil {
-		return fmt.Errorf("%s: %w", cmd.Use, err)
-	}
 	bp, err := fileRelative(args[0])
 	if err != nil {
 		return err
 	}
-	fmt.Println(bp)
-	t, err := topoNew(kubecfg, topopb, topo.WithBasePath(bp))
-	if err != nil {
-		return fmt.Errorf("%s: %w", cmd.Use, err)
-	}
-	out := cmd.OutOrStdout()
-	fmt.Fprintf(out, "Topology:\n%s\n", proto.MarshalTextString(topopb))
-	if err := t.Load(cmd.Context()); err != nil {
-		return fmt.Errorf("failed to load topology: %w", err)
-	}
-	if dryrun {
-		return nil
-	}
-	if err := t.Push(cmd.Context()); err != nil {
-		return err
-	}
-	if err := t.CheckNodeStatus(cmd.Context(), timeout); err != nil {
-		return err
-	}
-	fmt.Fprintf(out, "Topology %q created\n", topopb.Name)
-	r, err := t.Resources(cmd.Context())
-	if err != nil {
-		return fmt.Errorf("%s: %w", cmd.Use, err)
-	}
-	fmt.Fprintf(out, "Pods:\n")
-	for _, p := range r.Pods {
-		fmt.Fprintf(out, "%s\n", p.Name)
-	}
-
-	return nil
+	log.Infof(bp)
+	p := topo.TopologyParams {
```

Comment:
The New and Load functions are used to override the topo.New and topo.Load in the unit tests. For the Load function, I removed it since I can use another way to make the testing work.